src/tinuous/github.py

```diff
@@ -34,6 +35,12 @@
     sanitize_pathname,
 )
 
+RETRY_STATUSES = [500, 502, 503, 504]
```

Comment:
eh, I was trying to avoid "equating" the `RETRY_STATUSES` to all the statues we might want to skip on (only 5xxs).  ATM they are indeed identically but in general not necessarily the same, so IMHO it mixes the semantics abit ... but since functionally is ok ATM, let's proceed. Thank you @jwodder ! Merging